dotCMS/src/main/java/com/dotmarketing/business/ajax/HostAjax.java

```diff
@@ -24,51 +24,69 @@
 import com.liferay.portal.PortalException;
 import com.liferay.portal.SystemException;
 import com.liferay.portal.model.User;
-
-import org.quartz.SchedulerException;
-
 import java.io.File;
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
 import javax.servlet.http.HttpServletRequest;
+import org.quartz.SchedulerException;
 
 public class HostAjax {
 
 	private HostAPI hostAPI = APILocator.getHostAPI();
 	private UserWebAPI userWebAPI = WebAPILocator.getUserWebAPI();
 
 	public Map<String, Object> findHostsForDataStore(String filter, boolean showArchived, int offset, int count) throws PortalException, SystemException, DotDataException, DotSecurityException {
+		return findHostsForDataStore(filter, showArchived, offset, count, Boolean.FALSE);
+	}
+
+	public Map<String, Object> findHostsForDataStore(String filter, boolean showArchived, int offset, int count,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)